sample/kotlin/src/main/kotlin/com/datadog/android/sample/logs/LogsFragment.kt

```diff
@@ -61,17 +78,9 @@ internal class LogsFragment :
 
     override fun onClick(v: View) {
         interactionsCount++
-        when (v.id) {
-            R.id.log_warning -> logger.w("User triggered a warning")
-            R.id.log_error -> logger.e(
-                "User triggered an error",
-                IllegalStateException("Oops")
-            )
-            R.id.log_critical -> logger.wtf(
-                "User triggered a critical event",
-                UnsupportedOperationException("Oops")
```

Comment:
having exceptions sent was a nice scenario to test symbolication/error handling outside of crashes, should we maybe still send them?